packages/webapp/src/pages/collection.tsx

```diff
@@ -1,8 +1,62 @@
 import { type NextPage } from "next";
 import Head from "next/head";
 import { Navbar } from "../components/navbar";
+import React, { useState, useEffect } from 'react';
+import { useInventoryGetCollection } from "../generated";
```

Comment:
We haven't done this everywhere, but we should start:

1. Making all imports from files in the project start with `src` instead of being relative. (Sasha made this possible with the typescript config. So `src/generated.tsx` instead of `../generated`. One advantage is the import will stay correct if we move the file around.
2. Let's separate the imports from project files and the imports from files from libraries (e.g. start with all the library imports, then all the local file imports)
3. (less important) avoid semicolons — I will probably look into linters and enforce this automatically at some point